threads/models.py

```diff
@@ -61,15 +71,38 @@ class Comment(models.Model):
     updated_at = models.DateTimeField()
     content = models.TextField()
     trashed = models.BooleanField(default=False)
-    author = models.ForeignKey(Author, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
     thread = models.ForeignKey(Thread, on_delete=models.CASCADE)
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 
     def to_json(self) -> CommentJson:
         return {
             "id": self.id,
             "content": self.content,
-            "author": self.author.to_json(),
             "created_at": self.created_at.isoformat(timespec="seconds"),
             "updated_at": self.updated_at.isoformat(timespec="seconds"),
+            "user": self.user.to_json(),
+        }
+
+
+class UserReport(models.Model):
+    created_at = models.DateTimeField()
+    description = models.TextField()
+    category = models.CharField(max_length=200)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    thread = models.ForeignKey(Thread, on_delete=models.CASCADE)
+    comment = models.ForeignKey(
+        Comment, on_delete=models.CASCADE, blank=True, null=True
```

Comment:
Why can `comment` be blank? This doesn't match with e.g. `to_json` where you assume that comments exist (see also failing `yarn lint:types`